### PR TITLE
Added green button workshop

### PR DESCRIPTION
**Pull request by yixinwei-oracle**
1. All green button labs are under sandbox-labs, including one that is not used at this time (install-mgmt-agent-oca);
2. Green button index.html and manifest.json are under workshops/sandbox

Preview: [https://yixinwei-oracle.github.io/jms-livelabs-staging-developer/java-management/workshops/sandbox/index.html](https://yixinwei-oracle.github.io/jms-livelabs-staging-developer/java-management/workshops/sandbox/index.html)

The content is same as the previous preview link

@enhl-orc @sktruong-oracle

### CODE REVIEW

**Review by enhl-orc — APPROVED** [review]
submit the Self QA

**Comment by yixinwei-oracle** [comment]
Self QA done
[self_QA_sandbox_workshop.pdf](https://github.com/tchoo-oracle/jms-livelabs-staging-developer/files/9480254/self_QA_sandbox_workshop.pdf)

**Review by sktruong-oracle — APPROVED** [review]

File: java-management/workshops/sandbox/manifest.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        },
+        {
+            "title": "Lab 3: Perform Java Lifecycle Management with Java Management Service",
+            "filename":"../../sandbox-labs/life-cycle-management/life-cycle-management.md"
+
+        },
+        {
+            "title": "Need Help?",
+            "description": "Solutions to Common Problems and Directions for Receiving Live Help",
+            "filename": "https://objectstorage.us-phoenix-1.oraclecloud.com/p/SJgQwcGUvQ4LqtQ9xGsxRcgoSN19Wip9vSdk-D_lBzi7bhDP6eG1zMBl0I21Qvaz/n/c4u02/b/common/o/labs/need-help/need-help-livelabs.md"
```
Comment (tchoo-oracle) [inline java-management/workshops/sandbox/manifest.json:32]: @yixinwei-oracle  can you confirm whether the file should be from object store or oracle-livelabs.github.io?
image

File: java-management/workshops/sandbox/manifest.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    "help": "esther.neoh.hui.lin@oracle.com",
+    "tutorials": [
+        {
+            "title": "Introduction",
+            "filename": "../../sandbox-labs/introduction/introduction.md"
+        },
+        {
+            "title": "Get Started",
+            "description": "This is the prerequisites for customers using Free Trial and Paid tenancies, and Always Free accounts (if applicable). The title of the lab and the Contents Menu title (the title above) match for Prerequisite lab. This lab is always first.",
+            "filename": "https://objectstorage.us-phoenix-1.oraclecloud.com/p/SJgQwcGUvQ4LqtQ9xGsxRcgoSN19Wip9vSdk-D_lBzi7bhDP6eG1zMBl0I21Qvaz/n/c4u02/b/common/o/labs/cloud-login/cloud-login-livelabs.md"
```
Comment (tchoo-oracle) [inline java-management/workshops/sandbox/manifest.json:14]: same comment, the file should be from object store or oracle-livelabs.github.io?

File: java-management/workshops/sandbox/manifest.json [hunk 3]
```diff
@@ -2,7 +2,7 @@
     "workshoptitle": "Get Started with Oracle Cloud Infrastructure Java Management Service",
     "include": {
     },
-    "help": "livelabs-help-oci_us@oracle.com",
+    "help": "esther.neoh.hui.lin@oracle.com",
```
Comment (enhl-orc) [inline java-management/workshops/sandbox/manifest.json:5]: Is this suppose to switch to a personal email

File: java-management/sandbox-labs/install-management-agent-oca/install-management-agent-oca.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
+# Install Management Agent on your Managed Instance using OCI console
```
Comment (tchoo-oracle) [inline java-management/sandbox-labs/install-management-agent-oca/install-management-agent-oca.md:1]: Do we still need this markdown?

Comment (enhl-orc) [reply java-management/sandbox-labs/install-management-agent-oca/install-management-agent-oca.md:1]: Not used at present, keeping it for the future when the install script is no longer used and it returns to OCA.

File: java-management/workshops/sandbox/manifest.json (hunk 1 above)
Comment (yixinwei-oracle) [reply java-management/workshops/sandbox/manifest.json:32]: I will replace them with GitHub pages

**Comment by yixinwei-oracle** [comment]
Although self-QA points to cloud-login-livelabs2.md, but I've checked cloud-login-livelabs.md should be the right file

File: java-management/sandbox-labs/install-management-agent-oca/install-management-agent-oca.md (hunk 4 above)
Comment (yixinwei-oracle) [reply java-management/sandbox-labs/install-management-agent-oca/install-management-agent-oca.md:1]: Yes, this one is also up to date, and may be used in the future

**Pull request merged**